src/v/cloud_roles/signature.cc

```diff
@@ -13,6 +13,7 @@
 #include "base/vlog.h"
 #include "bytes/bytes.h"
 #include "cloud_roles/logger.h"
+#include "cloud_roles/request_reponse_helpers.cc"
```

Comment:
i realize this is fixed in a later commit, but please fix the git history. this won't be bisectable.